tests/e2e/config/bootstrap.js

```diff
@@ -122,6 +122,38 @@ function observeConsoleLogging() {
 	} );
 }
 
+/**
+ * Runs Axe tests when the block editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForBlockEditor() {
+	if ( ! await page.$( '.block-editor' ) ) {
+		return;
+	}
+
+	await expect( page ).toPassAxeTests( {
+		// Temporary disabled rules to enable initial integration.
+		// See: https://github.com/WordPress/gutenberg/pull/15018.
```

Comment:
This code was copied and pasted from this PR. The reason the tests are disabled are because of this PR. For reference, it makes more sense to link to the original PR here.

Reply:
Then I'd rather link to https://github.com/WordPress/gutenberg/issues/15452 where progress on that is being tracked, _and_ this PR.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3294/commits/78161f9d42a87c9ca636467daaa96705ee4540aa